benchmark/docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+version: '3'
+
+services:
+  prometheus:
+    network_mode: "host" # only needed if run locally
+    image: prom/prometheus:latest
```

Comment:
could we fix versions just in case (to current recent)? Sometimes a breaking change can sneak in and a `latest` flag just dumbly follows it. At the same time `latest` pulls the `latest` image when you start the stack and then does not update it unless you remove the image, can easilly end up with two ppl silently having two different versions trying to figure out what happened

Reply:
done